### PR TITLE
Introduce CronJob suspend parameter

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: openshift/cronJob-template.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ objects:
       app: thoth-core
   spec:
     schedule: '*/30 * * * *'
-    suspend: true
+    suspend: ${THOTH_SUSPEND_JOB}
```
Comment (goern) [inline openshift/cronJob-template.yaml:26]: do we need some " around here?!

**Review by goern — DISMISSED** [review]
"?

File: openshift/cronJob-template.yaml (hunk 1 above)
Comment (fridex) [reply openshift/cronJob-template.yaml:26]: I don't think so - I think it can be directly passed here.

Comment (fridex) [reply openshift/cronJob-template.yaml:26]: It has to be without quotes otherwise I get an error "unrecognized type: bool"

**Comment by fridex** [comment]
Accidentally included in #52

**Pull request closed without merging**